pkgs/by-name/gp/gpt-cli/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  stdenv,
+  python3,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "gpt-cli";
+  version = "0.3.2";
+  format = "pyproject";
+
+  SHELL = "${stdenv.shell}";
+
+  src = fetchFromGitHub {
+    owner = "kharvd";
+    repo = "gpt-cli";
+    rev = "v${version}";
```

Comment:
make sure this is `tag = "v${version}"`. Tag is pretty new and is preferred to rev.

Reply:
This still needs to be fixed